include/termcolor/termcolor.hpp

```diff
@@ -383,6 +383,134 @@ namespace termcolor
     }
 
 
+    inline
+    std::ostream& high_grey(std::ostream& stream)
```

Comment:
nit: Here and below, could you plase use the word `bright` instead of `high`? I think it reflects the color better and is also used to describe corresponding ANSI codes.